neural_compressor/experimental/pytorch_pruner/pruner.py

```diff
@@ -1,4 +1,4 @@
-"""pruner module."""
+"""Pruner."""
```

Comment:
weiwei to confirm w/ wenhua about the directory naming

Reply:
INC has a pruner, so use pytorch_pruner for now